openapi/p4pa-pu-sil.openapi.yaml

```
@@ -1289,6 +1287,12 @@ components:
           $ref: "#/components/schemas/Action"
         installment:
           $ref: "#/components/schemas/InstallmentDTO"
+    QueryPaymentStatusType:
+      type: string
+      enum:
+        - PAYMENT_ID
+        - NOTICE_NUMBER
+        - IUD
```

I see that with `payment_id` you mean the `installmentId`, how should the SIL be aware of such a field? do he knows it with the name of `payment_id`?
```suggestion
        - PAYMENT_ID
        - NOTICE_NUMBER
        - IUD
```

---

the `payment_id` corresponds to the legacy `idSession`, which would be the reference id of the immediate payment with which its status can be queried

---

ok, let's forget about the legacy API...
Just using the nativeApi, how should the SIL know about this value? there is an API which is returning it? (the native version of the API which actually is returning the sessionId) That's API, how is calling this parameter?
This `paymentId`, which we know to be the `installmentId`, should have the same name used on that API